rfcs/0000-c-unwind-and-project-group-charter.md

```diff
@@ -0,0 +1,496 @@
+- Feature Name: `simple_c_panic_abi`
+- Start Date: 2019-08-29
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+* First, to modify Rust functions with the "C" abi to abort on unwinding, as proposed
+  in [rust-lang/rust#52652].
+* Second, to introduce the "C unwind" ABI, with virtually all details
+  (but not quite all) left as "to be determined".
+    * The **intent** of this ABI, however, is to permit Rust panics to
+      be "translated" into a "native" unwinding mechanism, though
+      precisely which is dependent on the target platform or other
+      compiler options.
```

Comment:
Which compiler options do you have in mind here ? 

_I think_ that the "native" unwinding mechanism is part of the target ABI, i.e., part of the target triple. So there should be no way to change it via compiler options, doing so requires a completely different target. 

For example, on windows, the mingw targets specify the unwinding ABI as part of their triple in the MinGW C toolchains `...-mingw-seh`, `...-mingw-sjlj`, etc.